assets/stylesheets/components/_forms.scss

```diff
@@ -235,45 +235,85 @@ textarea {
     }
   }
 
-  input.form-control {
-    position: relative;
-    margin: 2px 0 0 2px;
-    z-index: 1;
-    height: 38px;
-    -webkit-box-flex: 1;
-    -ms-flex: 1;
-    flex: 1;
+  &.searchbar-addon {
+    input.form-control {
+      -webkit-box-flex: 1;
+      -ms-flex: 1;
+      flex: 1;
+      height: 34px;
+      margin: 2px;
+      position: relative;
+      top: 2px;
+      width: calc(100% - 4px);
+      z-index: 1;
+
+      // Removes validation styles
+      &.ng-touched {
+        &.ng-valid,
+        &.ng-invalid {
+          border-color: lighten(rgba(231,231,231,1), 5);
+          -webkit-box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
+          box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
+        }
+
+        &.ng-valid:focus,
+        &.ng-invalid:focus {
+          border-color: lighten(rgba(231,231,231,1), 5);
+          box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
+        }
+      }
 
-    // Removes validation styles
-    &.ng-touched {
-      &.ng-valid,
-      &.ng-invalid {
-        border-color: lighten(rgba(231,231,231,1), 5);
-        -webkit-box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
-        box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
+      &[placeholder] {
+        text-overflow: ellipsis;
       }
 
-      &.ng-valid:focus,
-      &.ng-invalid:focus {
-        border-color: lighten(rgba(231,231,231,1), 5);
-        box-shadow: inset 0 0 0 1px lighten(rgba(231,231,231,1), 5);
+      &:focus {
+        border-color: transparent;
+        box-shadow: none;
       }
-    }
 
-    &[placeholder] {
-      text-overflow: ellipsis;
+      &:focus + .searchbar-wrapper {
+        border: 1px solid rgba(139,206,237,1);
+        -webkit-box-shadow: inset 0 0 0 1px rgba(139, 206, 237, .5);
+        box-shadow: inset 0 0 0 1px rgba(139,206,237, .5);
+      }
     }
+  }
 
-    &:focus {
-      border-color: transparent;
-      box-shadow: none;
-    }
+  input.form-control {
+    margin-top: 0;
+    margin-bottom: 0;
 
-    &:focus + .searchbar-wrapper {
-      border: 1px solid rgba(139,206,237,1);
-      -webkit-box-shadow: inset 0 0 0 1px rgba(139, 206, 237, .5);
-      box-shadow: inset 0 0 0 1px rgba(139,206,237, .5);
-    }
+    // Removes validation styles
```

Comment:
@zarahzachz Do we need to keep all this commented code in here?

Reply:
Oops... pulling it out now.